neuro_san/http_sidecar/handlers/streaming_chat_handler.py

```diff
@@ -46,15 +40,13 @@ async def stream_out(self,
             return 0
 
         sent_out: int = 0
-        async for sub_generator in generator:
-            async for result_message in sub_generator:
-                result_dict: Dict[str, Any] = MessageToDict(result_message)
-                result_str: str = json.dumps(result_dict) + "\n"
-                self.write(result_str)
-                flush_ok = await self.do_flush()
-                if not flush_ok:
-                    return sent_out
-                sent_out += 1
+        async for result_dict in generator:
+            result_str: str = json.dumps(result_dict) + "\n"
+            self.write(result_str)
+            flush_ok = await self.do_flush()
+            if not flush_ok:
+                return sent_out
+            sent_out += 1
         return sent_out
```

Comment:
Now it is regular async generator, instead of curious case of generator-of-generators as we had before.